server/events/delete_lock_command.go

```diff
@@ -53,37 +53,22 @@ func (l *DefaultDeleteLockCommand) DeleteLocksByPull(repoFullName string, pullNu
 		return numLocks, err
 	}
 	if numLocks == 0 {
-		l.Logger.Debug("No locks found for pull")
+		l.Logger.Debug("No locks found for repo '%v', pull request: %v", repoFullName, pullNum)
 		return numLocks, nil
 	}
 
+	// The locks controller currently has no implementation of Atlantis project names, so this is hardcoded to an empty string.
+	projectName := ""
```

Comment:
if we set this to empty then the plan is not removed while running `atlantis unlock` and returns the following error:
```
failed to delete locks by pull remove /atlantis-data/repos/<path-to-the-project>/<workspace>.tfplan: no such file or directory
```
as the plan was created with the name `<project-name>-<workspace>.tfplan` whereas this tries to remove `<workspace>.tfplan` as the plan file name from here returns it:
https://github.com/runatlantis/atlantis/blob/4a921b3d658c84a901c99bbc2b6d822fbb067e9f/server/core/runtime/runtime.go#L92-L94